From fe31ff4eff1a0dc82e469363ae821a274031f415 Mon Sep 17 00:00:00 2001
From: dianlujitao <dianlujitao@lineageos.org>
Date: Wed, 21 Mar 2018 08:39:28 +0000
Subject: [PATCH 2/3] Revert "Don't enable ADB by default when ro.adb.secure is
 1"

 * This caused sys.usb.config set to none after boot, MTP and adb
   are both unusable.

This reverts commit 47c2f914bd3203431ed386e2b144bdcd9ac1cd28.

Change-Id: Icd16f2837f2479bfba164c88e32295ec684b15b0
---
 init/property_service.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/init/property_service.cpp b/init/property_service.cpp
index 5f34a16c4..25517edbd 100644
--- a/init/property_service.cpp
+++ b/init/property_service.cpp
@@ -659,11 +659,11 @@ static void load_persistent_properties() {
 // So we need to apply the same rule of build/make/tools/post_process_props.py
 // on runtime.
 static void update_sys_usb_config() {
-    bool is_secure = android::base::GetBoolProperty("ro.adb.secure", true);
+    bool is_debuggable = android::base::GetBoolProperty("ro.debuggable", false);
     std::string config = android::base::GetProperty("persist.sys.usb.config", "");
     if (config.empty()) {
-        property_set("persist.sys.usb.config", !is_secure ? "adb" : "none");
-    } else if (!is_secure && config.find("adb") == std::string::npos &&
+        property_set("persist.sys.usb.config", is_debuggable ? "adb" : "none");
+    } else if (is_debuggable && config.find("adb") == std::string::npos &&
                config.length() + 4 < PROP_VALUE_MAX) {
         config.append(",adb");
         property_set("persist.sys.usb.config", config);
-- 
2.15.1

